package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "builder": "~2.9.1",
     "builder-victory-component": "^2.3.0",
+    "d3-array": "^1.0.1",
```

Comment:
I prefer not to add this package. Can you use lodash `sum` and just use max and min instead of `extent`